ui/apps/dashboard/src/components/Onboarding/useOnboardingStep.ts

```diff
@@ -1,16 +1,44 @@
-import { useState } from 'react';
-import { useLocalStorage } from 'react-use';
+import { useEffect, useState } from 'react';
 
 import { STEPS_ORDER, type OnboardingSteps } from './types';
 
 export default function useOnboardingStep() {
-  const [onboardingLastStepCompleted, setOnboardingLastStepCompleted] =
-    useLocalStorage<OnboardingSteps>('onboardingLastStepCompleted', undefined);
-
+  // Temporary approach, we will store this value in the backend in the future
   const [lastCompletedStep, setLastCompletedStep] = useState<OnboardingSteps | undefined>(
-    onboardingLastStepCompleted
+    undefined
   );
 
+  useEffect(() => {
+    if (typeof window !== 'undefined') {
```

Comment:
Is this window check necessary? I'd code in useEffect would only ever run in the browser.